service/src/main/resources/application.properties

```diff
@@ -22,6 +22,9 @@ spring.datasource.password=123456
 spring.jpa.generate-ddl=true
 spring.datasource.driver-class-name=org.postgresql.Driver
 spring.jpa.show-sql=true
+# kerberos
+# add new configuration for kerberos file
+krb5conf.path=/path/to/krb5conf/file
```

Comment:
Should we comment this configuration by default ?

Reply:
Excuse me,I have a little confused,would you explain it for me?

Reply:
> Should we comment this configuration by default ?
Excuse me,I have a little confused,would you explain it for me?